app/assets/javascripts/actions/actionTypes.js

```diff
@@ -12,6 +12,7 @@ export default keyMirror({
   EDIT_STORY: null,
   UPDATE_STORY: null,
   UPDATE_STORY_SUCCESS: null,
+  SET_LOADING: null,
```

Comment:
Shouldn't we use something like `SET_LOADING_STORY` or `UPDATE_STORY_LOADING`

I suppose there will be more loading statuses in the application, so imagine another reducer listens to `SET_LOADING` as well, we might get some conflicts of interest 😂

Reply:
good idea, better safe than sorry! :smile: